app/views/users/show.html.erb

```diff
@@ -8,4 +8,13 @@
       </h1>
     </section>
   </aside>
+  <div class="col-md-8">
+    <% if @user.microposts.any? %>
```

Comment:
có thể dùng `@microposts` thay thế không

Reply:
k dc ạ, 
@user.microposts : ứng  với micropost với user người dùng.

Reply:
@hieungv chú chắc chưa 👿

Reply:
em thay the thi no chay loi. :)

Reply:
@microposts.any? thi dc anh a. :3